Fiction notes/Lists/GDL_PorosityDistribution_TodoList.md

```diff
@@ -0,0 +1,7 @@
+:memo: Lists 2019/4/15:
```

Comment:
Add to-dos to be checked later (refer to the timeline, it should be done before 4.19)